src/include/stir/DiscretisedDensity.h

```diff
@@ -112,6 +112,8 @@ template<int num_dimensions, typename elemT>
   typedef Array<num_dimensions,elemT> base_type;
   typedef DiscretisedDensity<num_dimensions,elemT> self_type;
 public:
+  typedef elemT pixel_type;
```

Comment:
remove. use `full_value_type` which is already there (from `Array`)

Reply:
Thanks, missed that